mods/farming/init.lua

```diff
@@ -7,7 +7,7 @@
 
```

Comment:
Farming has been edited to be slower, grow when no people are around, and to support seasons. This undoes all that.

Reply:
fixed.